jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/FloydWarshallShortestPaths.java

```diff
@@ -298,16 +310,6 @@ private void lazyCalculateMatrix()
                 }
```

Comment:
Is there a reason why we use Double.compare() in the lazyCalculateMatrix method? It seems more efficient (and adequate) to simply compare the literals, e.g. `if(edgeWeight < d[v_1][v_2])`, instead of using a comparator.